server/index.js

```diff
@@ -1,16 +1,30 @@
 import path from "path";
 import express from "express";
+import expressSession from "express-session";
 import { userRouter } from "./routes/user.js";
 import { fileURLToPath } from "url";
 import { dirname } from "path";
+import dotenv from "dotenv";
 
+dotenv.config();
```

Comment:
instead of 
```
import dotenv from "dotenv";
dotenv.config();
```
we can just do
 `import "dotenv/config";`

Reply:
Done